test/app.js

```diff
@@ -8,19 +8,6 @@ var mockery = require('mockery');
 describe('generator:app', function () {
   before(function () {
     mockery.enable({warnOnUnregistered: false});
-    mockery.registerMock('github', function () {
```

Comment:
I never used mockery before but from my understanding it only mocks `require`'s. In that case, how should I mock this object?